src/styles/header/header.scss

```diff
@@ -56,6 +85,39 @@
   }
 }
 
+.input-search-field-container-mobile {
+  @include desktop {
+    display: none;
+  }
+  @include large-desktop {
```

Comment:
this looks to be unnecessary style since it is already being applied from desktop size